scipy/_lib/_numpy_compat.py

```diff
@@ -726,7 +726,7 @@ def cov(m, y=None, rowvar=True, bias=False, ddof=None, fweights=None,
             fweights = np.asarray(fweights, dtype=float)
             if not np.all(fweights == np.around(fweights)):
                 raise TypeError(
-                    "fweights must be integer")
+                    "fweights must be an integer")
```

Comment:
This change looks incorrect. `fweights` can be a scalar or an array; "an integer" is correct for scalar but not for array. Can you undo this?

Reply:
You're right, I'll undo this change now.